docs/using/publishing-an-existing-project-to-github.md

```diff
@@ -1,5 +1,6 @@
 # Publishing an existing project to GitHub
 
+0. Clean solution directories, bin, obj, etc. or else these will be uploaded to Github
```

Comment:
That should be covered with a proper `.gitignore`. I think it would be better to mention that instead of a step that just state WHAT but doesn't explain WHY, nor HOW.

So maybe a note or warning "make sure to have properly set your `.gitignore` before adding the project" witha  link to GitHub's page about `.gitignore` (i.e. https://docs.github.com/en/github/using-git/ignoring-files).